### ISSUE TITLE
Sound stuttering at arch linux

### ISSUE THREAD

**Issue by saddy123** [NicolasGuilloux/blade-shadow-beta#41]
Hey, 

with my old Radeon HD 6850 the stream lagged too much to play.

Today I installed a new RX580 and the lagging is gone but now I have permanent sound stuttering which happend before only a few times and was fixable with reboots.

Now nothing helps, do u have any suggestion? I tried the stable client, because the screen is purple with the beta client. 
But the sound seems to be working, but it doesn't help if I can't see anything (screenshot: https://imgur.com/psaZtb8.png)

I ran the shadow-tools tool and alls was fine.
Im running an up to date arch linux with mesa driver.

**Comment by aarnoud** [comment]
Hi Saddy,

2 thing to try regarding your audio issue to start with when sound is stuttering:

-  `killall pulseaudio`
-   check/switch audio cable (Plug / unplug also jack)

**Comment by saddy123** [comment]
Hi aarnoud, 

both did not help.

I unplugged my headset 3.5mm + microphone and I unplugged + plugged my 3.5mm speakers. 
And I killed pulseaudio (then the sound died) but after restarting the stream I got the same behaviour. 

The stable and beta live images are also not working, while loading the shadow client the system shuts down.

I tried logging pulseaudio to debug but it doesn't seem to be helpful to me. 
Any tipps how to debug this? 
I mean the sound was working before the gpu change.. strange to me but gives me hope. 
[pulseverbose.log](https://github.com/NicolasGuilloux/blade-shadow-beta/files/3745156/pulseverbose.log)

**Comment by NicolasGuilloux** [comment]
Hello there!
Do you use the AppImage? Is your Shadow working fine on another device?
Can you give us the vainfo, just to check the compatibility?

**Comment by saddy123** [comment]
Hey, 

I tried both aur packages shadow-tech and shadow-beta and I tried running the AppImage manually.
Before I changed the gpu from radeon hd 6850 to rx580 the sound was finde with the stable client.

At my android device and my win10 Laptop its running good, but my goal was using shadow on my arch linux desktop. I am short before reinstalling the virtual machine.. 

Edit: Ok did it. Reinstalling/Resetting my shadow did not help. 
Deinstalling shadow-tech and shadow-beta and manually using the AppImage did not help too. :(

Edit@21-10-2019_06:30: 
Now its working. Dont ask me why but I hope it will stay that way
